matisse/src/main/java/com/zhihu/matisse/internal/ui/adapter/AlbumMediaAdapter.java

```diff
@@ -51,22 +51,24 @@
     private RecyclerView mRecyclerView;
     private int mImageResize;
 
-    public AlbumMediaAdapter(Context context, SelectedItemCollection selectedCollection, RecyclerView recyclerView) {
+    public AlbumMediaAdapter(Context context, SelectedItemCollection sC, RecyclerView rV) {
         super(null);
         mSelectionSpec = SelectionSpec.getInstance();
-        mSelectedCollection = selectedCollection;
+        mSelectedCollection = sC;
 
-        TypedArray ta = context.getTheme().obtainStyledAttributes(new int[]{R.attr.item_placeholder});
+        TypedArray ta = context.getTheme()
+                .obtainStyledAttributes(new int[]{R.attr.item_placeholder});
         mPlaceholder = ta.getDrawable(0);
         ta.recycle();
 
-        mRecyclerView = recyclerView;
+        mRecyclerView = rV;
     }
 
     @Override
     public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         if (viewType == VIEW_TYPE_CAPTURE) {
-            View v = LayoutInflater.from(parent.getContext()).inflate(R.layout.photo_capture_item, parent, false);
+            View v = LayoutInflater.from(parent.getContext()).
+                    inflate(R.layout.photo_capture_item, parent, false);
```

Comment:
Seems that if you set your IDE's max line length to be 120, this line should not wrap. The same below.